content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -246,50 +313,50 @@ Updating the installation is performed by rerunning the two commands listed in s
 
 1. To install kubectl on Windows you can use either [Chocolatey](https://chocolatey.org) package manager or [Scoop](https://scoop.sh) command-line installer.
 
-   {{< tabs name="kubectl_win_install" >}}
-   {{% tab name="choco" %}}
-   ```powershell
-   choco install kubernetes-cli
-   ```
-   {{% /tab %}}
-   {{% tab name="scoop" %}}
-   ```powershell
-   scoop install kubectl
-   ```
-   {{% /tab %}}
-   {{< /tabs >}}
+    {{< tabs name="kubectl_win_install" >}}
+    {{% tab name="choco" %}}
```

Comment:
How come the extra indentation?

Reply:
consistency--most indentation is 2 or 4 spaces on this page, so if I saw a 3 space indent I changed it.

Reply:
No. Three spaces indentation is the correct syntax. The intent is to have line 316 left aligned to the first "T" character on line 314. The markdown engine sometimes interpret 4 spaces indentation differently!!!

Reply:
I've made a mistake with the indentation -- I'll fix it throughout, thanks for pointing it out! :)